backend/events/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+# Create your models here.
+
+class Event(models.Model):
+    title = models.CharField(max_length=80)
```

Comment:
What's the logic behind the max length here?

Reply:
We figured since this was an event title, 80 characters would be sufficient enough. Do you recommend us to increase or decrease the value?

Reply:
done